cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -91,7 +91,6 @@ func LoadNetConf(bytes []byte) (*NetConf, logger.Logger, error) {
 
 	//logConfig
 	logConfig := logger.Configuration{
-		BinaryName:  conf.Name,
 		LogLevel:    conf.PluginLogLevel,
 		LogLocation: conf.PluginLogFile,
```

Comment:
The plugin will get right path passed in through `PluginLogFile`

Reply:
Yup, through aws-conf file :)